projects/igniteui-angular/src/lib/test-utils/grid-functions.spec.ts

```diff
@@ -1780,6 +1781,78 @@ export class GridFunctions {
         return headerTitle.parent;
     }
 
+    public static verifyGridPager(fix, rowsCount, firstCellValue, pagerText, buttonsVisibility) {
+        const grid = fix.componentInstance.grid;
+
+        expect(grid.getCellByColumn(0, 'ID').value).toMatch(firstCellValue);
+        expect(grid.rowList.length).toEqual(rowsCount, 'Invalid number of rows initialized');
+
+        if (pagerText != null) {
+            expect(grid.nativeElement.querySelector(PAGER_CLASS)).toBeDefined();
+            expect(grid.nativeElement.querySelectorAll('igx-select').length).toEqual(1);
+            expect(grid.nativeElement.querySelector('.igx-paginator__pager > div').textContent).toMatch(pagerText);
+        }
+        if (buttonsVisibility != null && buttonsVisibility.length === 4) {
+            const pagingButtons = GridFunctions.getPagingButtons(grid.nativeElement);
+            expect(pagingButtons.length).toEqual(4);
+            expect(pagingButtons[0].className.includes(BUTTON_DISABLED_CLASS)).toBe(buttonsVisibility[0]);
+            expect(pagingButtons[1].className.includes(BUTTON_DISABLED_CLASS)).toBe(buttonsVisibility[1]);
+            expect(pagingButtons[2].className.includes(BUTTON_DISABLED_CLASS)).toBe(buttonsVisibility[2]);
+            expect(pagingButtons[3].className.includes(BUTTON_DISABLED_CLASS)).toBe(buttonsVisibility[3]);
+        }
+    }
+
+    public static testPagingAPI(fix, grid: IgxGridComponent, page: (page: number) => void, desiredPageIndex: number = 2) {
```

Comment:
This method should be in the file grid.pagination.spec.ts.

Reply:
Done